src/pages/home/report/ReportActionsList.tsx

```diff
@@ -245,19 +236,18 @@ function ReportActionsList({
         }
 
         return null;
-    }, [accountID, sortedVisibleReportActions, unreadMarkerTime, messageManuallyMarkedUnread]);
+    }, [accountID, sortedVisibleReportActions, unreadMarkerTime]);
```

Comment:
```suggestion
    }, [sortedVisibleReportActions, unreadMarkerTime]);
```

Actually you should fix this